lib/Shipment/UPS.pm

```diff
@@ -453,90 +426,85 @@ The following service mapping is used:
   * express => 02 (UPS Second Day Air)
   * priority => 01 (UPS Next Day Air)
 
+This method ignores what is in $self->packages and uses a single package weighing 1 pound for rating. The idea is to list what services are available, but for accurate rate comparisons, the rate method should be used.
+
 =cut
 
 sub _build_services {
   my $self = shift;
 
   use Shipment::Package;
   use Shipment::Service;
-  use Shipment::UPS::WSDL::RateInterfaces::RateService::RatePort;
-  
-  my $interface = Shipment::UPS::WSDL::RateInterfaces::RateService::RatePort->new(
-    {
-      proxy_domain => $self->proxy_domain,
-    }
-  );
+
   my $response;
-  my %services;
 
-  if ($self->weight_unit ne 'oz')
```

Comment:
As per previous comment, this logic is part of adding back additional surepost options - see commit b56dba8c2f